commerce-notification-web/src/main/resources/META-INF/resources/view_notification_templates.jsp

```diff
@@ -74,9 +77,21 @@ portletURL.setParameter("notificationNavigationItem", notificationNavigationItem
 			</liferay-frontend:add-menu>
 		</c:if>
 	</liferay-frontend:management-bar-buttons>
+
+	<liferay-frontend:management-bar-action-buttons>
+		<liferay-frontend:management-bar-button
+			href='<%= "javascript:" + renderResponse.getNamespace() + "deleteCommerceNotificationTemplates();" %>'
```

Comment:
Couldn't we make this simpler? Like

    href="javascript:<%= renderResponse.getNamespace() %>deleteCommerceNotificationTemplates();"